sdk/src/fee.rs

```diff
@@ -146,10 +144,9 @@ impl FeeStructure {
             });
 
         FeeDetails {
-            transaction_fee: (signature_fee
+            transaction_fee: signature_fee
                 .saturating_add(write_lock_fee)
-                .saturating_add(compute_fee) as f64)
-                .round() as u64,
+                .saturating_add(compute_fee),
```

Comment:
needs a feature-gate 😢 f64 rounding could lead to a different result here if total fee is high....I think that this probably would not happen practically

Reply:
I guess safe to 2^53...so practically is this okay?